src/components/ErrorBoundary/index.jsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
-import NewRelicService from '../../data/services/NewRelicService';
+import { logError } from '@edx/frontend-platform/logging';
```

Comment:
@binodpant does this mean @raq929 also just fixed https://openedx.atlassian.net/browse/ENT-3852?  (*hopeful face*)

Reply:
@raq929 this may mean you should delete the service itself?

Reply:
yes I believe so! as long as we can confirm the new relic posting is covered in that logger call, seems like it should